codecov.yml

```diff
@@ -1,14 +1,26 @@
 # Enable coverage report message for diff on commit
+# Docs can be found here: https://docs.codecov.io/v4.3.0/docs/commit-status
 coverage:
   status:
-    project: off
+    project:
+      default:
+        target: auto
+        # Overall coverage should never drop more then 1%
+        threshold: 0.5
+        base: auto
+        branches: null
+        if_no_uploads: error
+        if_not_found: success
+        if_ci_failed: error
+        only_pulls: false
+        flags: null
+        paths: null
     patch:
       default:
-        # basic
         target: auto
-        threshold: 0.1
+        # Allows PRs without tests, overall stats count
+        threshold: 100
```

Comment:
Exactly. Patch level check will always be green. Otherwise if 1 code line is added which is not covered, it would have 0% coverage.